### PR TITLE
Add footer

### PR DESCRIPTION
**Pull request by benborgers**
TODO: 

- [ ] What content do we want in the footer? For now, I’ve just added the Glow logo that links to glow.app. 

2022-06-10 at 14 07@2x

### CODE REVIEW

**Comment by vpontis** [comment]
@benborgers maybe we can add a link to our Twitter on the right? We'll probably add some support / terms / etc links later on.

File: docs/icons/TwitterIcon.tsx [hunk 1]
```diff
@@ -0,0 +1,8 @@
+export const TwitterIcon = () => (
```
Comment (vpontis) [inline docs/icons/TwitterIcon.tsx:1]: Hmm, why are we doing this rather than using `svg`s?

File: docs/pages/_app.tsx [hunk 2]
```diff
@@ -111,13 +112,30 @@ export default function App({ Component, pageProps }: AppProps) {
             <NextPrev />
           </main>
         </div>
+
+        <footer>
```
Comment (vpontis) [inline docs/pages/_app.tsx:116]: Let's make the footer its own component that we import.

**Review by vpontis — APPROVED** [review]

File: docs/icons/TwitterIcon.tsx (hunk 1 above)
Comment (benborgers) [reply docs/icons/TwitterIcon.tsx:1]: Sure! I’ve set this PR into the state where I’m getting the error. In `<Footer />` I’m importing `<TwitterIcon />` and trying to use it, but I get: `Error: Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: object`.

https://github.com/glow-xyz/nftoken/blob/a10ce7bac4b18747f9c02e9ce0cd41d86d7e56a8/docs/components/Footer.tsx#L1-L20

Comment (vpontis) [reply docs/icons/TwitterIcon.tsx:1]: @benborgers pushed a fix in 08762d2a40d97fd9df1ae79335a753912245dbf7

Comment (benborgers) [reply docs/icons/TwitterIcon.tsx:1]: Thank you!

**Pull request merged**